src/main/resources/org/verapdf/rest/views/restclient.mustache

```diff
@@ -123,7 +123,11 @@
       </div>
     </form>
       <div>
-          <h4>Validation Profile Information</h4>
+          <h4>Disclaimer</h4>
+          <p>This demo service is based on the offical veraPDF Docker image <a href="https://hub.docker.com/r/verapdf/rest" target="_blank">veraPDF/rest</a> and is 
+          supposed to be used only for test purposes. More details on the configuration of this service and tweaking its parameters for production use can be found at 
```

Comment:
"and is supposed to be used only for test purposes."
->
"and is only intended to be used for testing."

Reply:
fixed